README.md

```diff
@@ -899,31 +905,31 @@ The Variable EXIV2\_PORT or EXIV2\_HTTP can be set to None to skip http tests.
 
 [TOC](#TOC)
 <div id="4-1">
-
-### 4.1 Running tests on a UNIX-like system
+### 4.1 Running tests on Unix-like systems
 
 You can run tests directly from the build:
 
 ```bash
 $ cmake .. -G "Unix Makefiles" -DEXIV2_BUILD_UNIT_TESTS=On 
-$ make
-... lots of output ...
-$ make tests
+... lots of output and build summary ...
+$ cmake --build .
 ... lots of output ...
+$ cmake --build . --target test
```

Comment:
`ctest` instead?

Reply:
I think I fixed that this morning on 0.27-maintenance.  That's when I realised "there's still a lot of work to be done here.  I'll focus on getting `0.27-maintenance` correct.  Then punt to 'main'.".  So, then I made the 'main' PR a draft.

We were boozing last night.  I'll run tomorrow morning, then I'll have fun fixing everything.  It's our son's wedding anniversary today, so they've gone to Bath for the weekend.

Reply:
Enjoy!